com.unity.ml-agents/Tests/Editor/PublicAPI/PublicApiValidation.cs

```diff
@@ -83,6 +83,30 @@ public override float[] Heuristic()
             }
         }
 
+        // Simple SensorComponent that sets up a StackingSensor
+        class StackingComponent : SensorComponent
+        {
+            public SensorComponent wrappedComponent;
```

Comment:
Where is this SensorComponent living? If it is attached to the same agent, won't we have duplicate sensor data?

Reply:
I commented below that this isn't meant to be practical, it's just an example for the purposes of making sure we have the right access in this test.

https://github.com/Unity-Technologies/ml-agents/blob/57548acfd0cb41e3ced1ff98713741c250f75487/com.unity.ml-agents/Tests/Editor/PublicAPI/PublicApiValidation.cs#L142